joomla/joomla-cms
# Bootstrapify gallery plugin

Since it was suggested in various comments so far to get rid of the fotorama JS and CSS in the custom gallery field

### Summary of Changes
Removes Fotorama media and changes the gallery plugin to use Bootstrap 2.3 thumbnails.
It adds also a new option to specify how many thumbnails it should show per row. This is limited to values that make sense in a Bootstrap scenario (1,2,3,4,6,12).

### Testing Instructions
* Set up a custom field of type gallery and assign to an article
* Check frontend

### Expected result
![bootstrap](https://cloud.githubusercontent.com/assets/1018684/23320559/8ec79c1c-fadb-11e6-90c3-07259981f141.PNG)


### Actual result
![fotorama](https://cloud.githubusercontent.com/assets/1018684/23320555/8c3f9c56-fadb-11e6-921a-2bd869896dbd.PNG)


### Documentation Changes Required
If there is already a documentation about this custom field, then it needs to be adjusted.
